classes/options-general.php

```diff
@@ -668,7 +668,7 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 
 													<tr valign="top">
 														<td><label
-																for="auto_locate"><?php _e( 'Auto-detect Location', 'simplemap' ); ?></label>
+																for="auto_locate"><?php _e( 'Auto-detect Location', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 141 characters

<!-- Comment by [SideCI](https://sideci.com) -->